tests/test_platforms_dummy.py

```diff
@@ -20,3 +21,35 @@ def test_dummy_execute_pulse_sequence():
     sequence = PulseSequence()
     sequence.add(platform.create_qubit_readout_pulse(0, 0))
     result = platform.execute_pulse_sequence(sequence, nshots=100)
+
+
+@pytest.mark.parametrize("average", [True, False])
+def test_dummy_single_sweep(average):
+    platform = Platform("dummy")
+    sequence = PulseSequence()
+    pulse = platform.create_qubit_readout_pulse(qubit=0, start=0)
+    for parameter in Parameter:
+        if parameter is Parameter.amplitude:
+            parameter_range = np.random.rand(10)
+        else:
+            parameter_range = np.random.randint(10, size=10)
+        sequence.add(pulse)
+        sweeper = Sweeper(parameter, parameter_range, [pulse])
+        platform.sweep(sequence, sweeper, average=average)
+
+
+@pytest.mark.parametrize("average", [True, False])
+def test_dummy_double_sweep(average):
+    platform = Platform("dummy")
+    sweepers = []
+    sequence = PulseSequence()
+    pulse = platform.create_qubit_readout_pulse(qubit=0, start=0)
+    sequence.add(pulse)
+    for _ in range(2):
+        for parameter in Parameter:
+            if parameter is Parameter.amplitude:
+                parameter_range = np.random.rand(10)
+            else:
+                parameter_range = np.random.randint(10, size=10)
+        sweepers.append(Sweeper(parameter, parameter_range, [pulse]))
+    platform.sweep(sequence, *sweepers, average=average)
```

Comment:
This test is not asserting anything...

Reply:
Here I mainly wanted to test that the `sweep` method was not raising any errors.
I will check if I can add some assertions

Reply:
I've added some assertions. By adding them I was able to spot a bug in the code.
Thanks for the suggestion @AleCandido